src/main/java/org/opentripplanner/middleware/bugsnag/BugsnagDispatcher.java

```diff
@@ -0,0 +1,161 @@
+package org.opentripplanner.middleware.bugsnag;
+
+import org.opentripplanner.middleware.bugsnag.response.Organization;
+import org.opentripplanner.middleware.bugsnag.response.Project;
+import org.opentripplanner.middleware.models.BugsnagEvent;
+import org.opentripplanner.middleware.models.BugsnagEventRequest;
+import org.opentripplanner.middleware.utils.FileUtils;
+import org.opentripplanner.middleware.utils.HttpUtils;
+import org.opentripplanner.middleware.utils.JsonUtils;
+
+import java.net.URI;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+
+import static org.opentripplanner.middleware.spark.Main.getConfigPropertyAsInt;
+import static org.opentripplanner.middleware.spark.Main.getConfigPropertyAsText;
+
+/**
+ * Class to provide Bugsnag information bu constructing required API calls to Bugsnag
+ */
+public class BugsnagDispatcher {
+
+    private static final String BUGSNAG_API_URL = "https://api.bugsnag.com";
+    private static final String BUGSNAG_API_KEY = getConfigPropertyAsText("BUGSNAG_API_KEY");
+    private static final String BUGSNAG_ORGANIZATION = getConfigPropertyAsText("BUGSNAG_ORGANIZATION");
+    private static final int BUGSNAG_REPORTING_WINDOW_IN_DAYS
+        = getConfigPropertyAsInt("BUGSNAG_REPORTING_WINDOW_IN_DAYS", 14);
+
+    private static Organization ORGANIZATION = null;
+    public static HashMap<String, Project> PROJECTS = new HashMap<>();
+
+    private static HashMap<String, String> BUGSNAG_HEADERS = new HashMap<>();
+
+    private static String EVENT_DATA_REQUEST_FILTER;
```

Comment:
Since this doesn't appear to be reading in a ton of data and in the event of us needing to customize calls for event data, I think we could create this object using the ObjectMapper class. Something like this:

```java
ObjectNode reportingWindowCondition = mapper.createObjectNode();
reportingWindowCondition.put("type", "eq");
reportingWindowCondition.put("value", BUGSNAG_REPORTING_WINDOW_IN_DAYS);

ArrayNode sinceFilters = mapper.createArrayNode();
sinceFilters.add(reportingWindowCondition);

ObjectNode filters = mapper.createObjectNode();
filters.put("event.since", sinceFilters);

ObjectNode request = mapper.createObjectNode();
request.put("filters", fitlers);
```

Reply:
Updated.